### DISCUSSION TITLE
Adding another Domain to Postal with Different SMTP Hostname: Potential Impact on Reputation

### DISCUSSION THREAD

**Discussion by emahdij** [postalserver/postal#3086] [Help with using Postal]
Hello everyone,

I have recently set up Postal with the domain name x.com and the SMTP hostname mx.x.com. I’m now looking to add another domain, x.net, to my Postal configuration. I understand that the hostname and mail domains will be different in this case.

My question is: Will this setup, where the SMTP hostname differs from the mail domain, have any impact on my email reputation or cause any other issues?

For context, I have already configured x.net with Mailgun, and I'm considering adding some configurations to send mail via Postal on x.net based on the guidelines provided in Mailgun's FAQ [here](https://documentation.mailgun.com/docs/mailgun/faq/faqs/#can-i-use-the-same-domain-name-for-mailgun-and-for-google-apps-or-another-email-server).

I’d appreciate any advice. Thank you!

**Comment by willpower232** [comment]
Postal is intended to handle multiple domains, it takes a white-label approach so you have one set of domains pointing directly at your server and then in the UI add other domains pointing at the domains which point at your server.

There should not be any issues from adding other domains unless you (or whoever you let use your Postal server) start sending bad emails.
